src/LibChorus/merge/xml/generic/Conflict.cs

```diff
@@ -29,7 +29,11 @@ public abstract class Conflict : IConflict, IEquatable<Conflict>
 		[Obsolete("Use TimeFormatWithTimeZone instead, as TimeFormatNoTimeZone produces incorrect results when used with DateTime.Now")]
 		public static string TimeFormatNoTimeZone = @"yyyy-MM-ddTHH:mm:ssZ";
 		public static string TimeFormatWithTimeZone = @"yyyy-MM-ddTHH:mm:ssK";
-
+		private static readonly Encoding Utf8Encoder = Encoding.GetEncoding(
+			"UTF-8",
+			new EncoderReplacementFallback(string.Empty), // Drop any problem characters
```

Comment:
I wonder if it would be better to output something like "?" or "(error)" instead of silently hiding the problem by removing the problem characters?
A very fancy way would be to implement our own `EncoderFallback` and `EncoderFallbackBuffer` classes that would output the code point of the problem character.